src/board/system76/common/common.mk

```diff
@@ -35,13 +35,13 @@ board-common-y += wireless.c
 # 3 - INFO
 # 4 - DEBUG
 # 5 - TRACE
-CFLAGS+=-DLEVEL=5
+CFLAGS+=-DLEVEL=0
 
 # Uncomment to enable debug logging over keyboard parallel port
 #CFLAGS+=-DPARALLEL_DEBUG
 
 # Uncomment to enable I2C debug on 0x76
-#CFLAGS+=-DI2C_DEBUGGER=0x76
+CFLAGS+=-DI2C_DEBUGGER=0x76
```

Comment:
So it can be used to enable debugging the host on SMBUS address 0x76? DO we have some connectors to hook up into the battery header? @mkopec

Reply:
That is the goal. We can solder jumper to the battery connector (we already have for one device, trying to get it working now).

Reply:
I have tried the 2mm pitch jumper wires on Monday actually and it doesn't work. The battery connector pins are just too thin (checked with multimeter if there is a connection). If you have something smaller and more tight, then maybe it has a chance.